featuretools/primitives/standard/cum_transform_feature.py

```diff
@@ -47,8 +47,9 @@ class CumCount(TransformPrimitive):
         [1, 2, 3, 4, 5, 6]
     """
     name = "cum_count"
-    input_types = [[Id], [Discrete]]
-    return_type = Numeric
+    input_types = [[ColumnSchema(semantic_tags={'foreign_key', 'category'})],
+                   [ColumnSchema(semantic_tags={'category'})]]
+    return_type = ColumnSchema(semantic_tags={'numeric'})
```

Comment:
should this be the "Integer" logical type, since values are countable and `NaN` values are counted?

Reply:
Yeah, that makes sense to me!